src/Integration/src/IntegrationBase/Channel/AbstractSubscribableChannel.cs

```diff
@@ -48,6 +48,7 @@ public virtual bool Subscribe(IMessageHandler handler)
             var added = Dispatcher.AddHandler(handler);
             if (added)
             {
+                Logger?.LogTrace("Channel '" + ServiceName + "' has " + handler.ServiceName + " subscriber(s).");
```

Comment:
Does this log entry make sense? Seems very close to the one below and might seem odd side by side with the Info log on the next line

Reply:
I'm sure it was ported that way and the difference is in the trace level the name of handler is written out otherwise just the count. Do you think we should combine and have the handler name at info level?